### ISSUE TITLE
Galaxy 101: Reference to asterisks

### ISSUE THREAD

**Issue by luke-c-sargent** [galaxyproject/training-material#2086]
tl;dr:
- asterisks are now check-boxes (needs updated text/visuals)
- default workflow creation from history makes all workflow step check-boxes checked, instead of all unchecked (making limiting the output harder, particularly in the case of large workflows).

Asterisks (a.k.a. 'snowflakes') have been part of Galaxy UI Workflow editing for some time, but have been replaced by check-boxes. They are better visual communication for 'show' vs 'don't show' but their visuals and behavior are slightly different. By default, all boxes are checked when creating a history as opposed to the old style of 'no asterisks selected, which equals all selected.'

This is different from the described behavior of asterisks in the [Galaxy 101](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-101/tutorial.html) tutorial, where creating a workflow from a history has all boxes checked; to follow the tutorial steps to disable all but the outputs, the user has to tediously de-select many intermediate boxes. checkboxes also exhibit the 'none == all' selection, so the behavior could be changed to match the description.

possible solutions:
- amend text/screenshots to reflect checkboxes, and the default state of 'all checked'
- change the workflow creation's default 'all entries checked' behavior to 'all entries unchecked'

**Comment by luke-c-sargent** [comment]
also mentioned in [Galaxy 101 for everyone](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.html)

**Comment by shiltemann** [comment]
thanks @luke-c-sargent, we should definitely update the tutorials. It is a shame that selecting only a few outputs involves so much more clicking now, but maybe a "deselect all" button could be added in the future orso?
